### PR TITLE
refactor(phrases): extract ui phrases 

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Extract the main-flow phrase from the `@logto/phrases` package.  Put it as new `@logto/ui-phrases`.

Update all the references accordingly. 

Also , update the import logic of `@logto/phrases` in `@logto/schemas` so it won't pack the whole `@logto/phrases` package in. 

## Testing

UT passed
@logto-io/eng

### CODE REVIEW

File: packages/schemas/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "@logto/connector-types": "^0.1.0",
     "@logto/phrases": "^0.1.0",
     "@logto/shared": "^0.1.0",
+    "@logto/ui-phrases": "^0.1.0",
```
Comment (gao-sun) [inline packages/schemas/package.json:50]: phrases-ui?

File: packages/schemas/src/foundations/jsonb-types.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Language } from '@logto/phrases';
 import { hexColorRegEx } from '@logto/shared';
+import { Language } from '@logto/ui-phrases/lib/types.js';
```
Comment (gao-sun) [inline packages/schemas/src/foundations/jsonb-types.ts:2]: https://stackoverflow.com/questions/30302747/root-directory-in-package-json
Nit: use this for shorter import

**Review by gao-sun — APPROVED** [review]
Kool

File: packages/ui-phrases/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# `@logto/ui-phrases`
```
Comment (gao-sun) [inline packages/ui-phrases/README.md:1]: Change here accordingly if you’d like to change the package name

File: packages/schemas/package.json (hunk 1 above)
Comment (simeng-li) [reply packages/schemas/package.json]: updated

File: packages/schemas/src/foundations/jsonb-types.ts (hunk 2 above)
Comment (simeng-li) [reply packages/schemas/src/foundations/jsonb-types.ts]: updated

File: packages/ui-phrases/README.md (hunk 3 above)
Comment (simeng-li) [reply packages/ui-phrases/README.md]: updated

**Pull request merged**